# Arrays of booleans
bool[] availability